checks/process.go

```diff
@@ -163,12 +163,16 @@ func formatCommand(fp *process.FilledProcess) *model.Command {
 		Root:   "",    // TODO
 		OnDisk: false, // TODO
 		Ppid:   fp.Ppid,
-		Pgroup: fp.Pgrp,
```

Comment:
Removing pgroup because (a) we don't make it visible anywhere, (b) it's not available on mac.